tests/feature/feature_console/deregister.pm

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Test the function of SUSEConnect to deregister a module
+# Maintainer: Yi Xu <yxu@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+
+sub run() {
+    select_console 'root-console';
+
+    my $count = script_output("SUSEConnect --status-text | grep -c \"^\\s*Registered\"", 200);
+    if ($count eq 0) {
+        die "There is no module registered";
+    }
+
+    # check if a certain module is registered, e.g. containers.
+    my $module  = "sle-module-containers";
+    my $abbrv   = "contm";
```

Comment:
Looking at the sister scenarios for different architectures this will most likely fail aarch64. See the SCC_ADDONS variable in e.g. [aarch64 "pcm,tcm"](https://openqa.suse.de/tests/1019322#settings), [ppc64le "asmm,contm,lgm,pcm,tcm,wsm"](https://openqa.suse.de/tests/1019396#settings), [zkvm "asmm,contm,lgm,pcm,tcm,wsm"](https://openqa.suse.de/tests/1019523#settings), [x86_64 "asmm,contm,lgm,pcm,tcm,wsm"](https://openqa.suse.de/tests/1019859#settings)

I recommend to define that based on the architecture, e.g.

```perl
my $arch = get_var('ARCH', '');
my %module = (
    aarch64 => 'sle-module-public-cloud'
);
my $module = $module{$arch} // 'sle-module-containers';
my %abbrv = (
    aarch64 => 'pcm'
);
my $abbrv = $module{$arch} // 'contm';
```

or if you like that more

```perl
my $arch = get_var('ARCH', '');
my $module = check_var('ARCH', 'aarch64') ? 'sle-module-public-cloud' : 'sle-module-containers';
my $abbrv = check_var('ARCH', 'aarch64') ? 'pcm' : 'contm';
```